protocols/kad/src/kbucket/key.rs

```diff
@@ -25,6 +25,8 @@ use sha2::digest::generic_array::{GenericArray, typenum::U32};
 use std::borrow::Borrow;
 use std::hash::{Hash, Hasher};
 
```

Comment:
```suggestion
```

I don't think we use the convention of separating crate level imports and external imports with a newline.

Reply:
NP - this is how rust-analyzer plugin in VS Code added it